### ISSUE TITLE
2024-12-12, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#18]
# Today's Arxiv Papers

(1) [Generative Semantic Communication: Architectures, Technologies, and Applications](http://arxiv.org/abs/2412.08642v1), 2024-12-11

(2) [Fast Prompt Alignment for Text-to-Image Generation](http://arxiv.org/abs/2412.08639v1), 2024-12-11

(3) [Multimodal Latent Language Modeling with Next-Token Diffusion](http://arxiv.org/abs/2412.08635v1), 2024-12-11

(4) [Synthetic Vision: Training Vision-Language Models to Understand Physics](http://arxiv.org/abs/2412.08619v1), 2024-12-11

(5) [Exploiting the Index Gradients for Optimization-Based Jailbreaking on Large Language Models](http://arxiv.org/abs/2412.08615v1), 2024-12-11

(6) [Preference Discerning with LLM-Enhanced Generative Retrieval](http://arxiv.org/abs/2412.08604v1), 2024-12-11

(7) [Empirical Measurements of AI Training Power Demand on a GPU-Accelerated Node](http://arxiv.org/abs/2412.08602v1), 2024-12-11

(8) [Leveraging Graph-RAG and Prompt Engineering to Enhance LLM-Based Automated Requirement Traceability and Compliance Checks](http://arxiv.org/abs/2412.08593v1), 2024-12-11

(9) [Advancing Single- and Multi-task Text Classification through Large Language Model Fine-tuning](http://arxiv.org/abs/2412.08587v1), 2024-12-11

(10) [TURBOATTENTION: Efficient Attention Approximation For High Throughputs LLMs](http://arxiv.org/abs/2412.08585v1), 2024-12-11

(11) [Automated Soap Opera Testing Directed by LLMs and Scenario Knowledge: Feasibility, Challenges, and Road Ahead](http://arxiv.org/abs/2412.08581v1), 2024-12-11

(12) [Can We Generate Visual Programs Without Prompting LLMs?](http://arxiv.org/abs/2412.08564v1), 2024-12-11

(13) [Underestimated Privacy Risks for Minority Populations in Large Language Model Unlearning](http://arxiv.org/abs/2412.08559v1), 2024-12-11

(14) [MaestroMotif: Skill Design from Artificial Intelligence Feedback](http://arxiv.org/abs/2412.08542v1), 2024-12-11

(15) [EMS: Adaptive Evict-then-Merge Strategy for Head-wise KV Cache Compression Based on Global-Local Importance](http://arxiv.org/abs/2412.08521v1), 2024-12-11

(16) [Bridging Relevance and Reasoning: Rationale Distillation in Retrieval-Augmented Generation](http://arxiv.org/abs/2412.08519v1), 2024-12-11

(17) [AltFS: Agency-light Feature Selection with Large Language Models in Deep Recommender Systems](http://arxiv.org/abs/2412.08516v1), 2024-12-11

(18) [Strategies and Challenges of Efficient White-Box Training for Human Activity Recognition](http://arxiv.org/abs/2412.08507v1), 2024-12-11

(19) [SuperCode: Sustainability PER AI-driven CO-DEsign](http://arxiv.org/abs/2412.08490v1), 2024-12-11

(20) [Multi-GraspLLM: A Multimodal LLM for Multi-Hand Semantic Guided Grasp Generation](http://arxiv.org/abs/2412.08468v1), 2024-12-11

(21) [TapeAgents: a Holistic Framework for Agent Development and Optimization](http://arxiv.org/abs/2412.08445v1), 2024-12-11

(22) [From Multimodal LLMs to Generalist Embodied Agents: Methods and Lessons](http://arxiv.org/abs/2412.08442v1), 2024-12-11

(23) [Assessing Personalized AI Mentoring with Large Language Models in the Computing Field](http://arxiv.org/abs/2412.08430v1), 2024-12-11

(24) [SwarmGPT-Primitive: A Language-Driven Choreographer for Drone Swarms Using Safe Motion Primitive Composition](http://arxiv.org/abs/2412.08428v1), 2024-12-11

(25) [Detecting Conversational Mental Manipulation with Intent-Aware Prompting](http://arxiv.org/abs/2412.08414v1), 2024-12-11

(26) [Learning to Reason via Self-Iterative Process Feedback for Small Language Models](http://arxiv.org/abs/2412.08393v1), 2024-12-11

(27) [SweetieChat: A Strategy-Enhanced Role-playing Framework for Diverse Scenarios Handling Emotional Support Agent](http://arxiv.org/abs/2412.08389v1), 2024-12-11

(28) [NyayaAnumana & INLegalLlama: The Largest Indian Legal Judgment Prediction Dataset and Specialized Language Model for Enhanced Decision Analysis](http://arxiv.org/abs/2412.08385v1), 2024-12-11

(29) [HyViLM: Enhancing Fine-Grained Recognition with a Hybrid Encoder for Vision-Language Models](http://arxiv.org/abs/2412.08378v1), 2024-12-11

(30) [SmolTulu: Higher Learning Rate to Batch Size Ratios Can Lead to Better Reasoning in SLMs](http://arxiv.org/abs/2412.08347v1), 2024-12-11

(31) [Large Language Models Still Face Challenges in Multi-Hop Reasoning with External Knowledge](http://arxiv.org/abs/2412.08317v1), 2024-12-11

(32) [Code LLMs: A Taxonomy-based Survey](http://arxiv.org/abs/2412.08291v1), 2024-12-11

(33) [Lachesis: Predicting LLM Inference Accuracy using Structural Properties of Reasoning Paths](http://arxiv.org/abs/2412.08281v1), 2024-12-11

(34) [Y-NQ: English-Yorùbá Evaluation dataset for Open-Book Reading Comprehension and Text Generation](http://arxiv.org/abs/2412.08279v1), 2024-12-11

(35) [LCFO: Long Context and Long Form Output Dataset and Benchmarking](http://arxiv.org/abs/2412.08268v1), 2024-12-11

(36) [Large Language Models for Scholarly Ontology Generation: An Extensive Analysis in the Engineering Field](http://arxiv.org/abs/2412.08258v1), 2024-12-11

(37) [TouchTTS: An Embarrassingly Simple TTS Framework that Everyone Can Touch](http://arxiv.org/abs/2412.08237v1), 2024-12-11

(38) [Model-Editing-Based Jailbreak against Safety-aligned Large Language Models](http://arxiv.org/abs/2412.08201v1), 2024-12-11

(39) [Magneto: Combining Small and Large Language Models for Schema Matching](http://arxiv.org/abs/2412.08194v1), 2024-12-11

(40) [Exploring Multidimensional Checkworthiness: Designing AI-assisted Claim Prioritization for Human Fact-checkers](http://arxiv.org/abs/2412.08185v1), 2024-12-11

(41) [Auto-Generating Earnings Report Analysis via a Financial-Augmented LLM](http://arxiv.org/abs/2412.08179v1), 2024-12-11

(42) [TextRefiner: Internal Visual Feature as Efficient Refiner for Vision-Language Models Prompt Tuning](http://arxiv.org/abs/2412.08176v1), 2024-12-11

(43) [Can Graph Neural Networks Learn Language with Extremely Weak Text Supervision?](http://arxiv.org/abs/2412.08174v1), 2024-12-11

(44) [Collaborative Hybrid Propagator for Temporal Misalignment in Audio-Visual Segmentation](http://arxiv.org/abs/2412.08161v1), 2024-12-11

(45) [Unseen Horizons: Unveiling the Real Capability of LLM Code Generation Beyond the Familiar](http://arxiv.org/abs/2412.08109v1), 2024-12-11

(46) [Doubly-Universal Adversarial Perturbations: Deceiving Vision-Language Models Across Both Images and Text with a Single Perturbation](http://arxiv.org/abs/2412.08108v1), 2024-12-11

(47) [Adversarial Vulnerabilities in Large Language Models for Time Series Forecasting](http://arxiv.org/abs/2412.08099v1), 2024-12-11

(48) [What You See Is Not Always What You Get: An Empirical Study of Code Comprehension by Large Language Models](http://arxiv.org/abs/2412.08098v1), 2024-12-11

(49) [Multilingual LLMs Inherently Reward In-Language Time-Sensitive Semantic Alignment for Low-Resource Languages](http://arxiv.org/abs/2412.08090v1), 2024-12-11

(50) [Using Large Language Models for Parametric Shape Optimization](http://arxiv.org/abs/2412.08072v1), 2024-12-11

(51) [DialogAgent: An Auto-engagement Agent for Code Question Answering Data Production](http://arxiv.org/abs/2412.08069v1), 2024-12-11

(52) [ContextModule: Improving Code Completion via Repository-level Contextual Information](http://arxiv.org/abs/2412.08063v1), 2024-12-11

(53) [Federated In-Context LLM Agent Learning](http://arxiv.org/abs/2412.08054v1), 2024-12-11

(54) [M2SE: A Multistage Multitask Instruction Tuning Strategy for Unified Sentiment and Emotion Analysis](http://arxiv.org/abs/2412.08049v1), 2024-12-11

(55) [Quantifying the benefits of code hints for refactoring deprecated Java APIs](http://arxiv.org/abs/2412.08041v1), 2024-12-11

(56) [Bootstrapping Heterogeneous Graph Representation Learning via Large Language Models: A Generalized Approach](http://arxiv.org/abs/2412.08038v1), 2024-12-11

(57) [Scalable, Validated Code Translation of Entire Projects using Large Language Models](http://arxiv.org/abs/2412.08035v1), 2024-12-11

(58) [TinyThinker: Distilling Reasoning through Coarse-to-Fine Knowledge Internalization with Self-Reflection](http://arxiv.org/abs/2412.08024v1), 2024-12-11

(59) [MAGIC: Mastering Physical Adversarial Generation in Context through Collaborative LLM Agents](http://arxiv.org/abs/2412.08014v1), 2024-12-11

(60) [Concept Bottleneck Large Language Models](http://arxiv.org/abs/2412.07992v1), 2024-12-11

(61) [Agents for self-driving laboratories applied to quantum computing](http://arxiv.org/abs/2412.07978v1), 2024-12-10

(62) [Distributed Gradient Descent with Many Local Steps in Overparameterized Models](http://arxiv.org/abs/2412.07971v1), 2024-12-10

(63) [HalluCana: Fixing LLM Hallucination with A Canary Lookahead](http://arxiv.org/abs/2412.07965v1), 2024-12-10

(64) [Forking Paths in Neural Text Generation](http://arxiv.org/abs/2412.07961v1), 2024-12-10

(65) [PAFFA: Premeditated Actions For Fast Agents](http://arxiv.org/abs/2412.07958v1), 2024-12-10

(66) [MAGE: A Multi-Agent Engine for Automated RTL Code Generation](http://arxiv.org/abs/2412.07822v1), 2024-12-10

(67) [A large language model-based approach to quantifying the effects of social determinants in liver transplant decisions](http://arxiv.org/abs/2412.07924v1), 2024-12-10

(68) [Asking Again and Again: Exploring LLM Robustness to Repeated Questions](http://arxiv.org/abs/2412.07923v1), 2024-12-10

(69) [Rethinking Emotion Annotations in the Era of Large Language Models](http://arxiv.org/abs/2412.07906v1), 2024-12-10

(70) [Low-Rank Correction for Quantized LLMs](http://arxiv.org/abs/2412.07902v1), 2024-12-10

(71) [Towards Foundation-model-based Multiagent System to Accelerate AI for Social Impact](http://arxiv.org/abs/2412.07880v1), 2024-12-10

(72) [BiMediX2: Bio-Medical EXpert LMM for Diverse Medical Modalities](http://arxiv.org/abs/2412.07769v1), 2024-12-10

(73) [Bayesian Optimization of Antibodies Informed by a Generative Model of Evolving Sequences](http://arxiv.org/abs/2412.07763v1), 2024-12-10

(74) [Zero-Shot ATC Coding with Large Language Models for Clinical Assessments](http://arxiv.org/abs/2412.07743v1), 2024-12-10

(75) [Granite Guardian](http://arxiv.org/abs/2412.07724v1), 2024-12-10

(76) [DriveMM: All-in-One Large Multimodal Model for Autonomous Driving](http://arxiv.org/abs/2412.07689v1), 2024-12-10

(77) [Privacy-Preserving Customer Support: A Framework for Secure and Scalable Interactions](http://arxiv.org/abs/2412.07687v1), 2024-12-10

(78) [TRIM: Token Reduction and Inference Modeling for Cost-Effective Language Generation](http://arxiv.org/abs/2412.07682v1), 2024-12-10

(79) [RAZOR: Sharpening Knowledge by Cutting Bias with Unsupervised Text Rewriting](http://arxiv.org/abs/2412.07675v1), 2024-12-10

(80) [Ask Humans or AI? Exploring Their Roles in Visualization Troubleshooting](http://arxiv.org/abs/2412.07673v1), 2024-12-10

(81) [FlexLLM: Exploring LLM Customization for Moving Target Defense on Black-Box LLMs Against Jailbreak Attacks](http://arxiv.org/abs/2412.07672v1), 2024-12-10

(82) [Automating Business Intelligence Requirements with Generative AI and Semantic Search](http://arxiv.org/abs/2412.07668v1), 2024-12-10

(83) [Searching for Structure: Investigating Emergent Communication with Large Language Models](http://arxiv.org/abs/2412.07646v2), 2024-12-10

(84) [TrojanWhisper: Evaluating Pre-trained LLMs to Detect and Localize Hardware Trojans](http://arxiv.org/abs/2412.07636v1), 2024-12-10

(85) [ChocoLlama: Lessons Learned From Teaching Llamas Dutch](http://arxiv.org/abs/2412.07633v1), 2024-12-10

(86) [OmniDocBench: Benchmarking Diverse PDF Document Parsing with Comprehensive Annotations](http://arxiv.org/abs/2412.07626v1), 2024-12-10

(87) [DRUM: Learning Demonstration Retriever for Large MUlti-modal Models](http://arxiv.org/abs/2412.07619v1), 2024-12-10

(88) [Adapting to Non-Stationary Environments: Multi-Armed Bandit Enhanced Retrieval-Augmented Generation on Knowledge Graphs](http://arxiv.org/abs/2412.07618v1), 2024-12-10

(89) [DiffSensei: Bridging Multi-Modal LLMs and Diffusion Models for Customized Manga Generation](http://arxiv.org/abs/2412.07589v1), 2024-12-10

(90) [Scaling Sequential Recommendation Models with Transformers](http://dx.doi.org/10.1145/3626772.3657816), 2024-12-10

(91) [Hyperband-based Bayesian Optimization for Black-box Prompt Selection](http://arxiv.org/abs/2412.07820v1), 2024-12-10

(92) [Automatic Database Configuration Debugging using Retrieval-Augmented Language Models](http://arxiv.org/abs/2412.07548v1), 2024-12-10

(93) [CoPrUS: Consistency Preserving Utterance Synthesis towards more realistic benchmark dialogues](http://arxiv.org/abs/2412.07515v1), 2024-12-10

(94) [Ontology-driven Prompt Tuning for LLM-based Task and Motion Planning](http://arxiv.org/abs/2412.07493v1), 2024-12-10

(95) [Intelligent System for Automated Molecular Patent Infringement Assessment](http://arxiv.org/abs/2412.07819v1), 2024-12-10

(96) [Dynamic Ensemble Reasoning for LLM Experts](http://arxiv.org/abs/2412.07448v1), 2024-12-10

(97) [Optimizing Alignment with Less: Leveraging Data Augmentation for Personalized Evaluation](http://arxiv.org/abs/2412.07429v1), 2024-12-10

(98) [Beyond Search Engines: Can Large Language Models Improve Curriculum Development?](http://dx.doi.org/10.1007/978-3-031-72312-4_17), 2024-12-10

(99) [Generating Knowledge Graphs from Large Language Models: A Comparative Study of GPT-4, LLaMA 2, and BERT](http://arxiv.org/abs/2412.07412v1), 2024-12-10

(100) [MoDULA: Mixture of Domain-Specific and Universal LoRA for Multi-Task Learning](http://arxiv.org/abs/2412.07405v1), 2024-12-10
